src/stats/_statswidget.py

```diff
@@ -187,25 +188,29 @@ def _injectCSS(self):
         if util.THEME.themeurl("ladder/style.css"):
             injectWebviewCSS(self.webview.page(), util.THEME.readstylesheet("ladder/style.css"))
 
+    def set_player(self, player):
+        if self.player != player:
+            self.player = player
+            self.loaded = False
```

Comment:
Optional nitpicks - self.player -> self.selected_player and self.loaded -> self.should_reload would document themselves a bit better IMO.